### PR TITLE
Laying Down

### PR DESCRIPTION
**Pull request by LankLTE**
I can't actually test this currently. PR is up so I can run tests & merge master after the upstream merge easier.

### CODE REVIEW

File: Content.Server/_CD/Traits/LayEmoteComponent.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+using Content.Shared.Chat.Prototypes;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server._CD.Traits;
+
+/// <summary>
+/// Set players' blood to coolant, and is used to notify them of ion storms
```
Comment (DEATHB4DEFEAT) [inline Content.Server/_CD/Traits/LayEmoteComponent.cs:8]: Seem to have forgotten to change this

File: Content.Server/_CD/Traits/LayEmoteComponent.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+/// </summary>
+[RegisterComponent, NetworkedComponent, Access(typeof(LayEmoteSystem))]
+public sealed partial class LayEmoteComponent : Component
+{
+    [DataField("layEmote", customTypeSerializer: typeof(PrototypeIdSerializer<EmotePrototype>))]
+    public string LayEmoteId = "LayDown";
+
+    [DataField("standEmote", customTypeSerializer: typeof(PrototypeIdSerializer<EmotePrototype>))]
+    public string StandEmoteId = "StandUp";
```
Comment (DEATHB4DEFEAT) [inline Content.Server/_CD/Traits/LayEmoteComponent.cs:17]: Shouldn't these be `ProtoId`?

File: Content.Server/_CD/Traits/LayEmoteSystem.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+using Content.Shared.Movement.Systems;
+using Content.Shared.Standing;
+
+namespace Content.Server._CD.Traits;
+
+public sealed class LayEmoteSystem : EntitySystem
+{
+    [Dependency] private readonly MovementSpeedModifierSystem _modifier = default!;
+    [Dependency] private readonly StandingStateSystem _standingSystem = default!;
+    [Dependency] private readonly SharedBodySystem _bodySystem = default!;
```
Comment (DEATHB4DEFEAT) [inline Content.Server/_CD/Traits/LayEmoteSystem.cs:14]:
```suggestion
    [Dependency] private readonly StandingStateSystem _standing = default!;
    [Dependency] private readonly SharedBodySystem _body = default!;
```

File: Resources/Prototypes/_CD/Voice/lay_emotes.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    - stands
+    - stands.
+    - standing
+    - standing.
+    - stand up
+    - stand up.
+    - stands up
+    - stands up.
+    - stood up
+    - stood up.
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/_CD/Voice/lay_emotes.yml:37]:
```suggestion
    - stood up.

```

File: Resources/Prototypes/_CD/Voice/lay_emotes.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/_CD/Voice/lay_emotes.yml:1]: Do you really have to add the variations with periods? That's pretty annoying.

File: Content.Server/_CD/Traits/LayEmoteComponent.cs (hunk 1 above)
Comment (LankLTE) [reply Content.Server/_CD/Traits/LayEmoteComponent.cs:8]: yea

File: Content.Server/_CD/Traits/LayEmoteSystem.cs (hunk 3 above)
Comment (LankLTE) [reply Content.Server/_CD/Traits/LayEmoteSystem.cs:14]: ?

File: Resources/Prototypes/_CD/Voice/lay_emotes.yml
Comment (LankLTE) [reply Resources/Prototypes/_CD/Voice/lay_emotes.yml:1]: afaik yes

File: Content.Server/_CD/Traits/LayEmoteComponent.cs (hunk 2 above)
Comment (LankLTE) [reply Content.Server/_CD/Traits/LayEmoteComponent.cs:17]: Most uses I can find upstream use string

**Pull request merged**